src/api/nodeRpcCall.js

```diff
@@ -1912,6 +1912,16 @@ const congressSendTreasuryLld = async (transferToAddress, transferAmount, wallet
   return await submitExtrinsic( extrinsic, walletAddress);
 }
 
+const createBlacklistMotion = async (proposalHash, referendumIndex, walletAddress) => {
```

Comment:
```suggestion
const congressDemocracyBlacklist = async (proposalHash, referendumIndex, walletAddress) => {
```

The naming convention for creating motions is `congress[SomeAction]`.